src/stratis_cli/_error_reporting.py

```diff
@@ -205,6 +205,7 @@ def handle_error(err):
     :param Exception err: an exception
     """
 
+    # pylint: disable=unnecessary-comprehension
```

Comment:
You can just use ```list(get_errors(err))```

Reply:
Oops! That was a rebase failure on my part. It was already resolved in master. I've rebased and removed that commit.